imports/ui/ledger/ledger.js

```diff
@@ -11,13 +11,25 @@ import bech32 from "bech32";
 import sha256 from "crypto-js/sha256"
 import ripemd160 from "crypto-js/ripemd160"
 import CryptoJS from "crypto-js"
-
+import { LedgerSigner } from "@cosmjs/ledger-amino";
+import { SigningCosmosClient, makeSignDoc } from "@cosmjs/launchpad";
+import { sleep } from "@cosmjs/utils";
+import { SigningStargateClient, assertIsBroadcastTxSuccess, defaultRegistryTypes, makeCosmoshubPath } from "@cosmjs/stargate"
+import { Registry } from '@cosmjs/proto-signing';
+import {
+    MsgDelegate,
+    MsgUndelegate,
+    MsgBeginRedelegate
+} from '@cosmjs/stargate/build/codec/cosmos/staking/v1beta1/tx';
+import { MsgSend } from '@cosmjs/stargate/build/codec/cosmos/bank/v1beta1/tx';
 // TODO: discuss TIMEOUT value
 const INTERACTION_TIMEOUT = 10000
 const REQUIRED_COSMOS_APP_VERSION = Meteor.settings.public.ledger.ledgerAppVersion || "2.16.0";
 const DEFAULT_DENOM = Meteor.settings.public.bondDenom || 'uatom';
 export const DEFAULT_GAS_PRICE = parseFloat(Meteor.settings.public.ledger.gasPrice) || 0.025;
 export const DEFAULT_MEMO = 'Sent via Big Dipper'
+const API = Meteor.settings.public.remote.api
+const RPC = Meteor.settings.public.remote.rpc
```

Comment:
@MonikaCat please add this change to default settings as well. Also should the backend read from the `public` values instead of the `private` values in the `settings.json`?

Reply:
Ok updated! I removed those constants from ledger.js as they are set globally in main.js file

Reply:
Ok! I moved remote part under public section and updated the ```default_settings.json``` file. Also I have set RPC and API as global constants in main.js